File: Ex20.c
#include<stdio.h>

int main(int argc, char const *argv[])
{
    // Inserção das variáveis

    float P1,P2,Media;

    // Inserção do primeiro valor
    printf("Insira a nota da P1: ");
    scanf("%f",&P1);

    
    // Inserção do segundo valor
    printf("Insira a nota da P2: ");
    scanf("%f",&P2);

    Media = (P1+(2*P2))/3;

    if (Media >= 5)
    {
        printf("Aprovado com a nota %f",Media);

    }
    else{

        printf("Reprovado com a nota %f",Media);
    }



    return 0;
}

File: Ex20.py
p1 = float(input("Insira a nota p1: "))
p2 = float(input("Insira a nota p2: "))

media = (p1 + (2*p2))/3

print(f"Sua media final foi de {media:.2f}.")

if media < 5:

    print("voce foi reprovado.")

else:

    print("Voce foi aprovado.")